spec/db/data/metabase_user_restrictions_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+describe "RestrictiveAccessForMetabaseUser" do
```

Comment:
This test file currently is in the `spec/db/data` folder right now, which is meant to be the specs for our data migrations

It probably belongs in the existing `spec/tasks/analytics_spec.rb` file with the other tests for this rake task